### DISCUSSION TITLE
Copy/Duplicate Saved Search Criteria and Result Lines

### DISCUSSION THREAD

**Discussion by c-forney** [burkybang/SuiteAdvanced-Feedback#117] [Ideas]
Hello! 

The ability to copy/duplicate Saved Search Criteria and Result Lines similar to the functionality of transaction lines would save a decent amount of time when building saved searches.

**Comment by burkybang** [comment]
Great idea! I'll check it out

**Comment by burkybang** [comment]
I finished this feature, and it will be available in the next release

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/c2e01dca-cad1-4166-873b-fb9f99825d8a)

**Comment by burkybang** [comment]
@c-forney It's out now!

**Reply by c-forney** [reply]
That's slick! Awesome Work! Thank you so much!
